models/LoginForm.php

```diff
@@ -93,22 +85,38 @@ function ($attribute) {
             ],
             'rememberMe' => ['rememberMe', 'boolean'],
         ];
+
+        if (!Yii::$app->getModule('user')->debug)
```

Comment:
Module is injected inside the model, and it will always be instance of Yii2-user module. You need to use `$this->module` not `Yii::$app->module`.